README.md

```diff
@@ -1,2 +1,13 @@
-# proposals
-EMQ X Enhancement Proposals
+# EMQ X Improvement Proposals (EIP)
```

Comment:
Do we need a new name? It looks like the EIP doesn't match the name of our repository - rfc.

Reply:
repo has been renamed to eip

Reply:
reasons:
1. RFC does not make much sense (in a general way), it was an arbitrary name which happened to have made its fame
2. we will often refer to a proposal with a number like EIP-1234, having to use RFC-1234 will cause confusion.